Training/python/2018v1/Training_v0p1.py

```diff
@@ -234,25 +248,44 @@ def run_training(train_suffix, model_name, model, data_loader, to_profile):
                          epochs = data_loader.n_epochs, initial_epoch = data_loader.epoch,
                          callbacks = callbacks)
 
-    model.save("%s_final.tf" % train_name, save_format="tf")
+    model_path = f"{path_to_model}/{log_name}_final.tf"
+    model.save(model_path, save_format="tf")
+    mlflow.log_artifacts(model_path, "model")
     return fit_hist
 
-
-config   = os.path.abspath( "../../configs/training_v1.yaml")
-scaling  = os.path.abspath("../../configs/ShuffleMergeSpectral_trainingSamples-2_files_0_50.json")
-dataloader = DataLoader.DataLoader(config, scaling)
-netConf_full, input_shape, input_types  = dataloader.get_config()
-
-n_cells_eta = dataloader.n_cells
-n_cells_phi = dataloader.n_cells
-n_outputs = dataloader.tau_types
-
-setup_gpu(dataloader)
-
-TauLosses.SetSFs(*dataloader.TauLossesSFs)
-print("loss consts:",TauLosses.Le_sf, TauLosses.Lmu_sf, TauLosses.Ltau_sf, TauLosses.Ljet_sf)
-model = create_model(netConf_full)
-
-compile_model(model, dataloader.learning_rate)
-fit_hist = run_training('step{}'.format(1), dataloader.model_name, model, dataloader, False)
-
+@hydra.main(config_path='.', config_name='train')
```

Comment:
If I understand it correctly, it can: `config_path` [specifies the relative location](https://hydra.cc/docs/advanced/search_path) of the yaml cfg with respect to the main script. However, I tried to run it from the different folder now and it doesn't work out-of-the-box because of the `sys.path.insert(0, "..")`. So this line should be changed accordingly to point to the directory with `common.py` and `DataLoader.py` modules.

Reply:
If mlflow part is flexible in relation to the current directory, it is good, thanks. Yes, it should be changed to `sys.path.insert(0, os.path.dirname(__file__)+"/..")`